app/config/AppConfig.scala

```diff
@@ -51,5 +51,6 @@ class AppConfig @Inject()(runModeConfiguration: Configuration, environment: Envi
   lazy val invitationCallbackUrl: String = s"$baseUrlPensionAdministrator${runModeConfiguration.underlying.getString("serviceUrls.invitation.callback")}"
 
   lazy val isManualIVEnabled: Boolean = runModeConfiguration.getBoolean("features.is-iv-enabled").getOrElse(false)
```

Comment:
Also, not sure you need to add to appconfig now. Only needs to go into featureswitchmanagementservice now

Reply:
removed as not required